src/client/client4.ts

```diff
@@ -1204,6 +1204,24 @@ export default class Client4 {
         );
     };
 
+    sendEmailInvitesToTeamGracefully = async (teamId: string, emails: string[]) => {
+        this.trackEvent('api', 'api_teams_invite_members', {team_id: teamId});
+
+        return this.doFetch(
+            `${this.getTeamRoute(teamId)}/invite/email?graceful=true`,
```

Comment:
Is `graceful=true` for backwards compatibility? Without this flag, when there is an invalid email in the bunch, do none of the emails get sent every time?

Reply:
That is correct :)